test/journal/update_journal_entry_test.js

```diff
@@ -105,6 +106,26 @@ describe("Journal", function () {
                 expect(response.body.mood).to.equal("");
             });
         });
+        //moodEmoji
+        it("rejects a blank mood Emoji", function () {
+            return updatePatientEntry({}, {
+                moodEmoji : ""
+            }).then(function (response) {
+                expect(response).to.be.an.api.error(400, "invalid_emoji");
+                expect(response.body.moodEmoji).to.equal("");
+            });
+        });
+        it("allows a null mood Emoji", function (){
+            return updatePatientEntry({}, {
+                moodEmoji: null
+            }).then(function (response) {
+                console.log(response.body.errors);
+                expect(response).to.be.a.journal.success;
```

Comment:
It looks as though this test is failing with an API error `invalid_emoji`. The test is right, so you'll need to take a look at the `journal_entry.js` model file and fix the code to make the test pass :)